x-pack/plugins/observability_solution/dataset_quality/public/components/flyout/degraded_fields/spark_plot.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiLoadingChart,
+  euiPaletteColorBlind,
+  useEuiTheme,
+} from '@elastic/eui';
+import React from 'react';
+import { ScaleType, Settings, Tooltip, Chart, BarSeries } from '@elastic/charts';
+import { i18n } from '@kbn/i18n';
+import { Coordinate } from '../../../../common/types';
+
+export function SparkPlot({
+  valueLabel,
+  isLoading,
+  series,
+}: {
+  valueLabel: React.ReactNode;
+  isLoading: boolean;
+  series?: Coordinate[] | null;
+}) {
+  return (
+    <EuiFlexGroup
+      justifyContent="flexStart"
+      gutterSize="xs"
+      responsive={false}
+      alignItems="flexEnd"
+    >
+      <EuiFlexItem grow={false}>
+        <SparkPlotItem isLoading={isLoading} series={series} />
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>{valueLabel}</EuiFlexItem>
```

Comment:
This text need to of of size `xs` (`12px`) to match the design. Also, can we use the same number formatter as elsewhere on the flyout?

![image](https://github.com/elastic/kibana/assets/2748376/a6b277c3-c5ab-4632-ab95-7aa5e36a056e)

Reply:
Good catch, will update it now

Reply:
<img width="764" alt="image" src="https://github.com/elastic/kibana/assets/7416358/6cbc2226-123d-46e1-ab5d-f5551da611e6">